solr/core/src/test/org/apache/solr/handler/admin/CoreAdminCreateDiscoverTest.java

```diff
@@ -112,32 +112,32 @@ public void testCreateSavesSysProps() throws Exception {
     // verify props are in persisted file
 
     Properties props = new Properties();
-    File propFile =
-        new File(solrHomeDirectory, coreSysProps + "/" + CorePropertiesLocator.PROPERTIES_FILENAME);
-    FileInputStream is = new FileInputStream(propFile);
-    try {
+    Path propFile =
+        solrHomeDirectory
+            .toPath()
+            .resolve(coreSysProps)
+            .resolve(CorePropertiesLocator.PROPERTIES_FILENAME);
+    try (InputStream is = Files.newInputStream(propFile)) {
```

Comment:
`Files.newBufferedReader`

Reply:
Addressed in 7079308294cb7f4d4c468dbdc890afd2a6d67d2e